src/store/login/login.reducer.js

```diff
@@ -120,7 +120,8 @@ function loginReducer (state = getInitialLoginState(), action) {
         case STEP_NAME.WALLET_LOADER: {
           return {
             ...state,
-            currentStep: state.steps[STEP_NAME.WALLET_LOADER].walletName === 'metaMask'
+            currentStep: state.steps[STEP_NAME.WALLET_LOADER].walletName === 'metaMask' ||
```

Comment:
I think that here instead of `metaMask` and `walletConnect` we can use the `WalletName` enum